deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/resources/migrationConsole.yaml

```diff
@@ -67,32 +66,13 @@ spec:
             - "/bin/sh"
             - "-c"
             - |
-    {{- if .Values.developerModeEnabled }}
-              source /.venv/bin/activate && pipenv install -e ~/lib/console_link
-    {{- end }}
```

Comment:
Thank you!  This is so much easier to follow and forces us to use the same systems that customers are using.